include/blas/device.hh

```diff
@@ -706,6 +706,18 @@ void Queue::work_ensure_size( size_t lwork )
 template <typename scalar_t>
 void shift_vec( int64_t n, scalar_t* v, scalar_t c, blas::Queue& queue );
 
+//------------------------------------------------------------------------------
+/// Conjugates n-element vector src and stores in dst
+///
+///
+
```

Comment:
Doxygen comments (`///`) should be on the definition rather than the prototype. I think they could be on _either_ prototype _xor_ definition, but should not be on _both_ because doxygen will get confused. (The doxygen comment on `shift_vec` above should not be there.)